### DISCUSSION TITLE
do you guys want plotting code?

### DISCUSSION THREAD

**Discussion by loftusa** [flatironinstitute/CaImAn#1247] [Show and tell]
I have code that makes the following from what is essentially an HDF file:

Movie with contours and nice-ish legend
<img width="540" alt="Screenshot 2023-06-13 at 11 56 05 AM" src="https://github.com/flatironinstitute/CaImAn/assets/12386450/1ffb0703-207a-40e1-8943-9271010c0d30">

Spike plot using estimates.S (separated, for us, by different recording periods)
<img width="636" alt="Screenshot 2023-06-13 at 11 56 55 AM" src="https://github.com/flatironinstitute/CaImAn/assets/12386450/ebc99616-eadd-44f1-87a3-7d9c52fa0531">

Plot for separate neurons with estimates.S as lines in the background, estimates.C in green, and the background residuals in blue
<img width="652" alt="Screenshot 2023-06-13 at 11 57 58 AM" src="https://github.com/flatironinstitute/CaImAn/assets/12386450/2cb11c57-be9a-4a53-8c5a-61e30d23383e">

Raster plot for neurons
<img width="658" alt="Screenshot 2023-06-13 at 11 59 07 AM" src="https://github.com/flatironinstitute/CaImAn/assets/12386450/9c696ffc-47bd-48e0-8280-8010bee2b002">

I can post the code here if you guys want, although it might need a bit of refactoring to get into caiman.

**Comment by EricThomson** [comment]
Definitely we'd love useful things that users might like, depending on the dependencies and some other factors (e.g., how hard will it be to maintain once you ride away into the sunset :) ).  It seems this would potentially be multiple useful visualization methods, and we'd probably want to discuss each one.

**Comment by zhouyi0812** [comment]
Could you please share your plotting code? Thank you!

**Comment by wmsheer** [comment]
I'd also be super interested in this plotting code for the time being if you're still willing to share. Thanks!

**Comment by AmadeusMJY** [comment]
I‘d appreciate it greatly if you can kindly share your plotting code! (maybe too long ago though) Thanks a lot!
